CHANGELOG.md

```diff
@@ -40,6 +40,8 @@ now hidden behind the feature flag `non-api`. Also, `Promise::from_parts` is mar
 - You can no longer create an `Robj` from a reference `&T`, where `T` is an `extendr`-impl. [[#759]](https://github.com/extendr/extendr/pull/759)
 - You can no longer use `from_robj`, as the trait `FromRobj` as been removed. Instead, use `try_from`.
 - It is no longer possible to access an R integer vector as a `&[u32]` [[#767]](https://github.com/extendr/extendr/pull/767)
+- It is no longer possible to generate bindings as part of the compilation of
+extendr. Feature `non-api` is broken.
```

Comment:
Can we be more specific with this? What do you mean `non-api` is broken? Isn't it that it will cause warnings when used with R-devel (R >=  4.5) and may not be guaranteed to work in the future.

Reply:
It does not work at all. Using it will not compile extendr. So either we remove all `non-api`-dependent code, or I don't know what to do.

I'm leaning towards deleting everything related to `non-api`.

Reply:
No, I think we keep it and back-fill it at a later point with libR-sys generated bindings. Once the C API is actually "stabilized" we can port everything over and then remove the non-api entirely.

Reply:
Good! So this is resolved.. Or?